spec/factories/withdrawal_factory.rb

```diff
@@ -2,6 +2,7 @@
 
 FactoryBot.define do
   factory :withdrawal do
-    explanatory_note { SecureRandom.alphanumeric }
+    public_explanation { SecureRandom.alphanumeric }
+    association :status, state: :withdrawn
```

Comment:
I don't think we want this status association here as it's not the owner of status. 

Consequences of this are:

When we do `FactoryBot.create(:status, :withdrawn)` a Status (id: 1) is created, then a Withdrawal is created via this factory, which then creates it's own Status (id: 2). So we end up with an extra status and Withdrawal being pointed at by 2 objects.

Simplest suggestion would be to just remove this from this factory, whereas more complex approaches would be to use the `after(:build)` block or to battle with `inverse_of` - but I doubt we're using this currently.